docs/theory/force_calibration/active.rst

```diff
@@ -91,3 +91,94 @@ hydrodynamically correct model for the peak:
 We can also include a distance to the surface like before. This results in an expression for the drag
 coefficient :math:`\gamma` that depends on the distance to the surface which is given by the same
 equations as listed in the section on the :doc:`hydrodynamically correct model<hyco>`.
+
+.. _bead_bead_theory:
+
+Bead-bead coupling
+^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+    The implementation of the coupling correction models is still alpha functionality.
+    While usable, this has not yet been tested in a large number of different scenarios.
+
+The active calibration method presented in the previous sections relies on oscillating the nanostage with a known amplitude and frequency.
+The fluid in the flow-cell follows the stage motion.
+This in turn exerts a drag on the bead that leads to a sinusoidal displacement of the bead from the trap center.
+The amplitude of the detected displacement (measured in Volts) and the stage amplitude are then quantified.
+From the stage amplitude (measured in microns, since the stage position is calibrated) an expected bead displacement is calculated.
+
+When using two beads, the flow field around the beads is reduced (because the presence of the additional bead slows down the fluid).
+The magnitude of this effect depends on the bead diameter, distance between the beads and their orientation with respect to the fluid flow.
+Streamlines for some bead configurations are shown below (simulated using FEniCSx :cite:`the_fenics_project_developers_2023_10432590`).
+
+.. image:: figures/streamlines.png
+  :nbattach:
+
+As a result, the bead moves less than expected for a given stage motion.
```

Comment:
Do you have an idea of how large the error can be when the bead-bead coupling is not accounted for? Is this discussed in one of the papers you are citing?

Reply:
The error depends strongly on the bead-bead distance to diameter ratio. I have added a small section on the expected error if you don't take it into account.

![errors](https://github.com/user-attachments/assets/870b24da-5a36-4d04-86dc-7bf9cf05ef2a)